test/over_react/util/rem_util_test.dart

```diff
@@ -205,7 +205,7 @@ main() {
           'correctly dispatches an event in resopnse to the first change', () async {
         expect(querySelector('#rem_change_sensor'), isNull);
 
-        var calls = [];
+        List<double> calls = [];
```

Comment:
#supernit Same here: should this be 
```dart
var calls = <double>[]?
```

Asking about these out mostly to improve my own knowledge.